manifest.yaml

```diff
@@ -16,6 +16,14 @@ arguments:
     description: Enables pre-releasing via semantic-release
     schema:
       type: boolean
+  ciOptions.skipE2e:
+    description: Disables the e2e ci pipeline
+    schema:
+      type: boolean
```

Comment:
I assume implicit default is `false` for stencil booleans?

Reply:
Yes, it's just depending on the Go default value behavior.